Makefile.include

```diff
@@ -205,8 +205,13 @@ export PREFIX ?= $(if $(TARGET_ARCH),$(TARGET_ARCH)-)
 # Add standard include directories
 INCLUDES += -I$(RIOTBASE)/core/include -I$(RIOTBASE)/drivers/include -I$(RIOTBASE)/sys/include
 INCLUDES += -I$(RIOTCPU)/$(CPU)/include
+CPUVENDOR ?= $(RIOTCPU)/$(CPU)/include/vendor
+ifneq (0,$(shell test -d $(CPUVENDOR)/; echo $$?))
```

Comment:
Can you use make's wildcard function here?

Reply:
mhm sure, if you give a hint what and how ... thx!

Reply:
To check for folder existance:
https://www.gnu.org/software/make/manual/html_node/Wildcard-Function.html

Reply:
shelling out is always more expensive than using make's natives

Reply:
to be honest: I simply copied from L169:

```
ifneq (0,$(shell test -d $(RIOTBOARD)/$(BOARD); echo $$?))
```